sources/org.osbuild.skopeo

```diff
@@ -103,13 +107,17 @@ class SkopeoSource(sources.SourceService):
         digest = image["digest"]
         tls_verify = image.get("tls-verify", True)
         transport = image.get("containers-transport", DOCKER_TRANSPORT)
+        location = image.get("storage-location", "")
 
         with tempfile.TemporaryDirectory(prefix="tmp-download-", dir=self.cache) as tmpdir:
             archive_dir = os.path.join(tmpdir, "container-archive")
             os.makedirs(archive_dir)
             os.chmod(archive_dir, 0o755)
 
-            reference = f"{imagename}@{digest}"
+            # skopeo will use the default storage location
+            # from `storage.conf`
```

Comment:
How does this relate to it using storage.conf? Maybe expand the comment a bit so that it's clear for someone not familiar with skopeo.

Reply:
Thanks, great feedback. Hopefully it's not too verbose now

Reply:
Hrm, possibly so :) Could it be boiled down to:
Skopeo will read the default storage path from the `/etc/containers/storage.conf` unless `storage-location` is provided.

If that's true then that tells me what's expected to happen with and without storage-location set.

Reply:
Haha thanks that's correct. Will sort it out